src-misc/locale/galaxy42_installer/pl.po

```diff
@@ -380,3 +380,41 @@ msgstr ""
 #, sh-format
 msgid "L_language"
 msgstr "Language=(PL)"
+
+msgid "L_install_option_sudo_cap_users_title"
+msgstr "Sudo dla cap-tool: wybór użytkowników/grup"
+
+msgid "L_install_option_sudo_cap_users"
+msgstr ""
+"Wybierz użytkowników i/lub grupy które będą mogłyużywać narzędzia "
+"cap-tool aby dać swoim programom prawa CAPtakie jak CAP_NET_ADMIN (zmiana "
+"ustawień sieci).\n"
+"Zazwyczaj daje się to zaufanemu użytkownikowi, który będzie kompilować "
+"nasz program.\n"
+"Oddzielając kolejne wyrazy znakiem spacji, podaj nazwy użytkowników,\n"
+"można też podać nazwy grup dopisując znak % na początku,na przykład "
+"tak:\n"
+"%sudo %admin %developers alice"
+
+msgid "L_warning_sudo_cap_install_failed_title $packages_str"
+msgstr "Sudo dla cap-tool: błąd"
+
+msgid "L_warning_sudo_cap_install_failed"
+msgstr ""
+"Sudo dla cap-tool: wystąpił błąd automatycznej konfiguracji sudo "
+"(sudoers).Możesz skonfigurować to ręcznie przez komendę visudo zamiast "
+"tego."
+
+msgid "L_install_option_sudo_cap_allow_title"
+msgstr "Sudo dla cap-tool: zgadzasz się?"
+
+msgid "L_install_option_sudo_cap_allow"
+msgstr ""
+"Proponujemy zainstalować i skonfigurować narzędzie cap-tool,\n"
+"tak aby zwykły użytkowników (bez root) mógł skompilować nasz program i "
```

Comment:
"i" \n ?

The end of lines are controlled by text output anyway (or by hard \n).

Not needed